operator/controllers/suite_test.go

```diff
@@ -125,7 +126,8 @@ var _ = BeforeSuite(func(done Done) {
 	//apiServerFlags = append(apiServerFlags, "--admission-control=MutatingAdmissionWebhook")
 
 	testEnv = &envtest.Environment{
-		CRDDirectoryPaths: []string{filepath.Join("..", "config", "crd", "bases"),
+		CRDDirectoryPaths: []string{
+			//filepath.Join("..", "config", "crd", "bases"),
```

Comment:
I can see you added the `SeldonDeployment` CRD to `../testing` as well. For future changes to the CRDs, would `kubebuilder` also write to that location?

Reply:
yes - not done automatically. We need the kustomize created version of the CRD. Will add to Makefile to generate.

Reply:
done